pact/Pact/Core/IR/Eval/Runtime/Utils.hs

```diff
@@ -373,3 +374,117 @@ anyCapabilityBeingEvaluated
 anyCapabilityBeingEvaluated caps = do
   capsBeingEvaluated <- use (esCaps . csCapsBeingEvaluated)
   return $! any (`S.member` caps) capsBeingEvaluated
+
+readKeyset' :: i -> Text -> EvalM e b i (Maybe KeySet)
```

Comment:
I would have not looked for it in this module, is there some reason for this reorg

Reply:
Good question

Initially I made some more aggressive changes to the gas model which actually impacted these functions,and I noticed the duplication across `Evaluator.hs` and `CoreBuiltin.hs`